tvtk/wrapper_gen.py

```diff
@@ -1336,3 +1482,291 @@ def %(setter)s(self, arg):
+        return re.match('|'.join(cls.special_traits.keys()),
+                        full_name) is not None
+
+    def _write_special_trait(self, klass, out, vtk_attr_name):
+        key = self._get_special_matched_key(klass, vtk_attr_name)
+        _, _, method_name = self.special_traits[key]
+        getattr(self, method_name)(klass, out, vtk_attr_name)
+
+    def _write_any_output(self, klass, out, vtk_attr_name):
+        self._write_get_output_method(klass, out, set=True)
+
+    def _write_any_source(self, klass, out, vtk_attr_name):
+        self._write_get_source_method(klass, out)
+
+    def _write_any_scalar_type(self, klass, out, vtk_attr_name):
+        """ This method does nothing """
+        pass
+
+    def _write_any_input(self, klass, out, vtk_attr_name):
+        self._write_get_input_method(klass, out)
+
+    def _write_any_input_connection(self, klass, out, vtk_attr_name):
+        self._write_get_input_connection_method(klass, out)
+
+    def _write_any_something_file_name(self, klass, out, vtk_attr_name):
+        name = self._reform_name(vtk_attr_name)
+        vtk_set_meth = getattr(klass, 'Set' + vtk_attr_name)
+
+        t_def = 'tvtk_base.vtk_file_name("")'
+        self._write_trait(out, name, t_def, vtk_set_meth, mapped=False)
+
+    def _write_any_something_file_prefix(self, klass, out, vtk_attr_name):
+        name = self._reform_name(vtk_attr_name)
+        vtk_set_meth = getattr(klass, 'Set' + vtk_attr_name)
+        t_def = 'tvtk_base.vtk_file_prefix("")'
+        self._write_trait(out, name, t_def, vtk_set_meth, mapped=False)
+
+    def _write_image_reader2_header_size(self, klass, out, vtk_attr_name):
+
+        if vtk_attr_name != 'HeaderSize':
+            raise RuntimeError("Not sure why you ask for me! "
+                               "I only deal with HeaderSize. Panicking.")
+
+        default, _ = self.parser.get_get_set_methods()[vtk_attr_name]
+
+        # The documentation of vtkImageReader2.GetHeaderSize says
+        # it returns `int`, but really the API meant `long`
+        # matters for Python 2
+        if PY_VER < 3:
+            default = long(default)
+            t_def = ('traits.Long({default}, '
+                     'enter_set=True, auto_set=False)').format(default=default)
+        else:
+            t_def = ('traits.Int({default}, '
+                     'enter_set=True, auto_set=False)').format(default=default)
+
+        # trait name
+        name = self._reform_name(vtk_attr_name)
+
+        # vtk set method
+        vtk_set_meth = getattr(klass, 'Set'+vtk_attr_name)
+
+        self._write_trait(out, name, t_def, vtk_set_meth, mapped=False)
+
+    def _write_hardware_selector_prop_color_value(self, klass, out,
+                                                  vtk_attr_name):
+
+        if vtk_attr_name != 'PropColorValue':
+            raise RuntimeError("Not sure why you ask for me! "
+                               "I only deal with PropColorValue. Panicking.")
```

Comment:
This is to prevent this method being used with `vtk_attr_name` other than `PropColorValue`.  This may be unnecessary given the method is private and is used internally.